README.md

```diff
@@ -46,5 +46,18 @@ Proxy options can be passed via envrionment variables.
  - `BROWSERSTACK_FORCE_PROXY` - if it's not empty, forces all traffic of Browserstack local binary to go through the proxy,
  - `BROWSERSTACK_FORCE_LOCAL` - if it's not empty, forces all traffic of Browserstack local binary to go through the local machine
 
+## Browserstack JS Testing and Browserstack Automate 
+Browserstack offers two APIs for browser testing:
+ - [Browserstack JS Testing](https://www.browserstack.com/javascript-testing-api)
+ - [Browserstack Automate](https://www.browserstack.com/automate)
+ 
+ JS testing supports more types of devices (compare: [JS Testing devices](https://www.browserstack.com/list-of-browsers-and-platforms?product=js_testing) 
+ vs [Automate devices](https://www.browserstack.com/list-of-browsers-and-platforms?product=automate)), 
+ while Automate allows for much longer tests ([2 hours](https://www.browserstack.com/automate/timeouts) vs [30 minutes](https://github.com/browserstack/api#timeout300)) 
+ and provides some additional features (like video recording on real mobile devices).
```

Comment:
Maybe we need to remove this phrase about video recording in light of what @pulkitsharma07 said